tests/client_test.go

```diff
@@ -17,7 +17,7 @@ func GetNewClient(t *testing.T) (*sdk.Lumos, context.Context) {
 		t.Fatalf(`LUMOS_API_TOKEN is not set`)
 	}
 	security := shared.Security{
-		HTTPBearer: &apiToken,
+		HTTPBearer: apiToken,
```

Comment:
This was leading to an error during compile testing. It appears we are attempting to dereference a pointer, when the value is actually a string literal